helpdesk-maker-checker-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/makerchecker/model/check/Checker.java

```diff
@@ -31,4 +31,6 @@
 
   Set<SymMessage> makeApprovedMessages(MakerCheckerMessage makerCheckerMessage, SymMessage symMessage);
 
+  void removeAttachmentAndDirectory(SymAttachmentInfo symAttachmentInfo);
```

Comment:
Please rename it to 'afterSendApprovedMessage' and provide the message object instead of the attachment.

Reply:
Done.

Reply:
Done.